sdk/devtestlabs/Azure.ResourceManager.DevTestLabs/samples/Generated/Samples/Sample_DevTestLabResource.cs

```diff
@@ -247,7 +247,7 @@ public async Task GenerateUploadUri_LabsGenerateUploadUri()
         public async Task ImportVm_LabsImportVirtualMachine()
         {
             // Generated from example definition: specification/devtestlabs/resource-manager/Microsoft.DevTestLab/stable/2018-09-15/examples/Labs_ImportVirtualMachine.json
-            // this example is just showing the usage of "Labs_ImportVm" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "Labs_ImportVirtualMachine" operation, for the dependent resources, they will have to be created separately.
```

Comment:
`ImportVirtualMachine` is the original operation id: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/devtestlabs/resource-manager/Microsoft.DevTestLab/stable/2018-09-15/DTL.json#L7873